.gitignore

```diff
@@ -27,3 +27,4 @@ pkg
 # RVM files
 .ruby-version
 .ruby-gemset
+/vendor
```

Comment:
Might be helpful to add a section for VS Code as well, as many devs would use it.

```
# VSCode #
.vscode/*
```

Reply:
I have added it now. Thanks Rahul

Reply:
Thanks for adding it.

Just a note that .vscode/* is mentioned under # RVM files. 🙈 

Should probably have a section of its own: `## VSCode` 😄

Reply:
its now under `#VScode` comment :D